k-distribution/include/kframework/builtin/README.md

```diff
@@ -20,5 +20,5 @@ the path prefix).
 -   [rat](rat.md): Rational number representation.
 -   [substitution](substitution.md): Hooked implementation of capture-aware
     sustitution for K definitions.
--   [unification](unification.k): Hooked implementation of unification
+-   [unification](https://github.com/kframework/k/blob/master/k-distribution/include/kframework/builtin/unification.k): Hooked implementation of unification
```

Comment:
This should be `.md`? Also any reason this is an absolute path?

Reply:
@nishantjr please open a new issue or a PR instead.